src/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/GenerateMonoDroidIncludes.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.IO;
+using System.Globalization;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tools.BootstrapTasks
+{
+	public class GenerateMonoDroidIncludes : Task
+	{
+		[Required]
+		public ITaskItem [] SourceFiles { get; set; }
+
+		[Required]
+		public ITaskItem [] DestinationFiles { get; set; }
+
+		public override bool Execute ()
+		{
+			Log.LogMessage (MessageImportance.Low, "Task " + nameof(GenerateMonoDroidIncludes));
```

Comment:
Please use `$"Task {nameof (GenerateMonoDroidIncludes)}"` instead of using `+`. :-)